.github/workflows/update-rest-api-json.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        branch: [main, 7.x, 7.14]
+        branch: ['main', '7.x', '7.14', '7.15']
```

Comment:
can you change this to a pattern, otherwise we'll have to remember this for every release?

for example: 
```
 branches:
      - main
      - master
      - '[0-9]+.[0-9]+'
      - '[0-9]+.x'
```

^^ copied from https://github.com/elastic/elasticsearch-net/blob/master/.github/workflows/test-jobs.yml#L13

Reply:
I'll open a follw-up for this, unfortunately, this is not github actions syntax, is the input for the nodejs script that downloads the artifacts.